src/main/resources/schema/ans/0.5.8/traits/trait_publishing.json

```diff
@@ -96,40 +118,43 @@
           "format": "date-time"
         },
 
-        "first_published_date": {
-          "title": "First Published Date",
+        "edition_first_publish_date": {
+          "title": "First Published Date (Edition)",
           "description": "The machine-generated date that this edition was created for the first time (i.e., that the content item was first published.)",
           "type": "string",
           "format": "date-time"
         },
 
-        "display_date": {
+        "edition_display_date": {
           "title": "Display Date (Edition)",
           "description": "The human-editable date that should be shown to readers as the 'date' for this content item. When viewing the story at this edition name directly, this will override whatever value is set for Display Date on the story directly. After an edition is created, subsequent updates to that edition will not change this date unless otherwise specified.",
           "type": "string",
           "format": "date-time"
         },
 
-        "publish_date": {
+        "edition_publish_date": {
           "title": "Publish Date (Edition)",
           "description": "The machine-editable date that should be shown to readers as the 'publish date' for this content item. When viewing the story at this edition name directly, this will override whatever value is set for Publish Date on the story directly. Every time an edition is updated (i.e. a story is republished) this date will also be updated unless otherwise specified.",
           "type": "string",
           "format": "date-time"
         },
 
-        "published": {
+        "edition_publish_status": {
```

Comment:
Since this is now namespaced, should it go back to `edition_published`?  Now that I look, that probably makes more sense for a boolean value - it's not that clear what `edition_publish_status=true` means.